src/reactivity/reactive.ts

```diff
@@ -143,7 +143,7 @@ export function shallowReactive<T extends object = any>(obj: T): T
 export function shallowReactive(obj: any): any {
   if (!isObject(obj)) {
     if (__DEV__) {
-      warn('"shallowReactive()" is called without provide an "object".')
+      warn('"shallowReactive()" is called without providing an "object".')
```

Comment:
I think we could use this opportunity to improve the whole wording:

```suggestion
      warn('"shallowReactive()" must be called on an object.')
```

@antfu @pikax WDYT?

Reply:
This sounds good to me